docs/core/versions/selection.md

```diff
@@ -0,0 +1,110 @@
+---
+title: .NET Core version selection
+description: Learn How .NET Core finds and chooses runtime versions for your program.
+author: billwagner
+ms.author: wiwagn
+ms.date: 06/27/2018
+---
+# .NET Core version selection
+
+[!INCLUDE [topic-appliesto-net-core-2plus](../../../includes/topic-appliesto-net-core-2plus.md)]
+
+This article explains the policies used by the .NET Core tools, SDK, and runtime for selecting versions. These policies provide a balance between running applications using the specified versions and enabling ease of upgrading both developer and end user machines. These policies perform the following:
+
+- Easy and efficient deployment of .NET Core, including security and reliability updates.
+- Usage of the latest tools and commands independent of target runtime.
+
+There are four activities during development and deployment where version selection occurs:
+
+- When you [run an SDK command](#the-sdk-uses-the-latest-installed-version).
+- When you [build an assembly](target-framework-monikers-define-build-time-apis).
+- When you [run a .NET Core application](framework-dependent-apps-roll-forward).
+- When you publish a [self-contained application](self-contained-deployments-include-the-selected-runtime).
+
+The rest of this document examines those four scenarios.
+
+## The SDK uses the latest installed version
+
+SDK commands include `dotnet new`, `dotnet build` or `dotnet run`. The `dotnet` CLI must choose an SDK version for any command. The .NET Core CLI uses the latest SDK installed on the machine. You'll use the .NET Core SDK v2.1.301 when it's installed, even if the project you are working with targets the .NET Core Runtime 2.0. Note that this is true for preview versions as well as released versions. You can take advantage of the latest SDK features and improvements while targeting earlier .NET Core runtime versions. You can target multiple runtime versions of .NET Core on different projects, using the same SDK tools for all projects.
+
+On rare occasions, you may need to use a different version of the SDK. You specify that version in a [global.json file](../tools/global-json.md). The "use latest" policy means you only use `global.json` to specify a .NET Core version earlier than the latest installed version.
```

Comment:
Although it's clarified in the second sentence, I'd change "different version" to "earlier version"